app/screens/permalink/index.js

```diff
@@ -43,7 +43,7 @@ function makeMapStateToProps() {
         }
 
         return {
-            channelId: channel ? channel.id : '',
+            channel,
```

Comment:
wow!!! why this? I believe the other prop you need is the channel Type and if it was deleted (archived), so why not just pass that instead of the whole object?

Reply:
Don't need type just deleted. What's your specific objection to using the object?

Reply:
Passing the whole object will cause the component to re-render even if it does have to, for instance you'll get a new post on that channel from the WS and there is no need to update the current View as is a permalink view but you are causing a re-render anyway

Reply:
Ah I see; yes I agree the attribute from the object would be better.